sources/packages/backend/libs/integrations/src/services/ssh/sftp-integration-base.ts

```diff
@@ -254,7 +262,7 @@ export abstract class SFTPIntegrationBase<DownloadType> {
    * Generates a new connected SFTP client ready to be used.
```

Comment:
Adding here for something related to the method `getResponseFilesFullPath`.

If we want to list only directories under a parent directory, this option can be used to be safer.

<img width="720" height="408" alt="Image" src="https://github.com/user-attachments/assets/b0751b9c-e7fb-42de-a0a5-02dcac2c9e8b" />

Reply:
It was implemented and then reverted.
I can add it as an optional parameter.

Reply:
Created an optional parameter to enforce directory type.